core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -328,6 +328,8 @@
   USER_LOCAL_WRITER_PACKET_SIZE_BYTES(Name.USER_LOCAL_WRITER_PACKET_SIZE_BYTES, "64KB"),
   USER_NETWORK_NETTY_CHANNEL(Name.USER_NETWORK_NETTY_CHANNEL, null),
   USER_NETWORK_NETTY_TIMEOUT_MS(Name.USER_NETWORK_NETTY_TIMEOUT_MS, 30000),
+  USER_NETWORK_NETTY_WRITER_CLOSE_TIMEOUT_MS(Name.USER_NETWORK_NETTY_WRITER_CLOSE_TIMEOUT_MS,
+      30000),
```

Comment:
I was thinking that the user can customize it if they need. Do you think this should be default to a high value? Most of the users will use Alluxio with object store?